bc_obps/registration/models.py

```diff
@@ -402,6 +407,12 @@ class Statuses(models.TextChoices):
     history = HistoricalRecords(table_name='erc_history"."operator_history', m2m_fields=[documents, contacts])
 
     class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["cra_business_number"],
+                name="cra_business_number_unique_constraint",
+            )
+        ]
```

Comment:
Could we add a test into this model to prevent unintentional removal of this constraint in the future?